compiler/tests-common/tests/org/jetbrains/kotlin/codegen/debugInformation/AbstractDebugTest.kt

```diff
@@ -120,6 +123,11 @@ abstract class AbstractDebugTest : CodegenTestCase() {
         super.tearDown()
     }
 
+    override fun updateConfiguration(configuration: CompilerConfiguration) {
+        super.updateConfiguration(configuration)
+        configuration.forceEnableFeatures(LanguageFeature.CorrectSourceMappingSyntax)
```

Comment:
Why not enable this feauture in test data explicitly?

Reply:
I thought it would be weird to require tests to opt into a bug fix in order to get the complete output.

Reply:
There are not so much test with inline. Otherwise we a getting additional complexity in test infrastructure, `forceEnableFeatures` and 'LanguageFeatureOverride' also looks a bit `weird`

Reply:
On a related note, if I'm reading this code https://github.com/JetBrains/kotlin/blob/master/compiler/tests-common/tests/org/jetbrains/kotlin/codegen/CodegenTestCase.java#L199-L212 correctly, tests tagged as `COMMON_COROUTINES_TEST` ignore the `!LANGUAGE` directive when running in experimental coroutine mode.

Reply:
oh...it's fail // cc @ilmirus

Reply:
LanguageFeatureOverride would fix this, just saying...

Reply:
@pyos thank you for point for this problem, let's keep LanguageFeatureOverride. Thank you!

Reply:
(－‸ლ)